how-to-guides/create-delete-projects/notebooks/Create_delete_projects.ipynb

```diff
@@ -102,10 +106,12 @@
    "source": [
     "from neptune import management\n",
     "\n",
-    "management.create_project(name=\"<WORKSPACE/PROJECT>\", \n",
-    "                          key=\"AMA\", \n",
-    "                          api_token=\"<YOUR_API_TOKEN>\", \n",
```

Comment:
In the case of project management, we did not want to show users that ANONYMOUS is an option and stay on the abstract level: "your api token"